seqr/views/apis/staff_api.py

```diff
@@ -1031,9 +1031,9 @@ def _update_variant_inheritance(variant, affected_individual_guids, unaffected_i
 def _get_genotype_zygosity(genotype):
     num_alt = genotype.get('numAlt')
     cn = genotype.get('cn')
-    if num_alt == 2 or cn == 0 or cn > 3:
+    if (num_alt != None and num_alt == 2) or (cn != None and (cn == 0 or cn > 3)):
```

Comment:
If `num_alt == 2` it can never be `None` and similarly  if `cn == 0 or cn > 3` than it can't be `None`, so this change won't do anything but is harder to read

Reply:
In Python 3,  '>' not supported between instances of 'NoneType' and 'int'.
Python 3:
```
>>> cn=None
>>> type(cn)
<class 'NoneType'>
>>> cn>2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'NoneType' and 'int'
``` 
Python 2:
```
>>> cn=None
>>> type(cn)
<type 'NoneType'>
>>> cn>2
False
```
But None instance does support `==`. I can remove `num_alt != None` checking.